cortex.proto

```diff
@@ -13,6 +13,11 @@ service Ingester {
   rpc LabelValues(LabelValuesRequest) returns (LabelValuesResponse) {};
   rpc UserStats(UserStatsRequest) returns (UserStatsResponse) {};
   rpc MetricsForLabelMatchers(MetricsForLabelMatchersRequest) returns (MetricsForLabelMatchersResponse) {};
+
+  // Claim allows leaving ingester to stream chunks directly to joining ingesters.
+  rpc ClaimStart(ClaimStartRequest) returns (ClaimStartResponse) {};
+  rpc ClaimStream(stream TimeSeriesChunk) returns (ClaimStreamResponse) {};
+  rpc ClaimFinish(ClaimFinishRequest) returns (ClaimFinishResponse) {};
```

Comment:
Rename to `Transfer{StartStreamFinish}`?

Also, can the `{Start,Finish}` methods be avoided?